Makefile.am

```diff
@@ -445,7 +445,7 @@ test:
 	./util/fi_info
 
 rpm: dist
-	LDFLAGS=-Wl,--build-id rpmbuild -ta libfabric-$(PACKAGE_VERSION).tar.bz2
+	LDFLAGS=-Wl,--build-id rpmbuild -ta $(RPMBUILD_OPTS) libfabric-$(PACKAGE_VERSION).tar.bz2
```

Comment:
nit: This specific change is unrelated to the provider and should be a separate patch.

Reply:
I'll pull it out